platform/bind/panel/packages/list.js

```diff
@@ -11,7 +11,11 @@ platform.core.node({
   outputs : ['list'],
   controlOutputs: [ platform.conventions.controls._Unauthorized ],
 }, (inputs, output, control) => {
-  platform.call(`${config.path}config/load`,
+  platform.call(
+    {
+      path: `${config.path}config/load`,
+      method: 'GET'
```

Comment:
shouldn't get be the implied method?

Reply:
I will take this into consideration in a next commit. I wanted to have an agnostic node type ("any") but that might not be the best approach. What do you think?